core/src/main/java/com/novoda/noplayer/drm/provision/HttpPoster.java

```diff
@@ -1,12 +1,13 @@
 package com.novoda.noplayer.drm.provision;
 
 import java.io.ByteArrayOutputStream;
+import java.io.DataOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.net.HttpURLConnection;
 import java.net.URL;
 
-class HttpPoster {
+public class HttpPoster {
```

Comment:
We could move the `HttpPoster` to a `support` package, it's not really relevant to the library anyway.